packages/frontend/amp/components/Ad.tsx

```diff
@@ -37,6 +37,36 @@ const adStyle = css`
     }
 `;
 
+const adClass = css`
+    display: none;
+`;
+
+const usAdRegionClass = css`
+    .amp-geo-group-us & {
+        display: block;
+    }
+`;
+
+const auAdRegionClass = css`
+    .amp-geo-group-au & {
+        display: block;
+    }
+`;
+
+const rowAdRegionClass = css`
+    .amp-geo-group-eea & {
+        display: block;
+    }
+`;
+
+const regionClasses = {
+    US: usAdRegionClass,
+    AU: auAdRegionClass,
+    ROW: rowAdRegionClass,
+};
+
+type Region = 'US' | 'AU' | 'ROW';
```

Comment:
Might be better to call type `AdRegion` to distinguish it from other kinds of region.
```suggestion
type AdRegion = 'US' | 'AU' | 'ROW';
```

Reply:
Agree. Changed in new commit. Pushing it next.